R/01.3-dados_setores_censitarios.R

```diff
@@ -238,13 +238,14 @@ merge_renda_setores_all <- function(ano, munis = "all") { # munis <- sigla <- "f
   # aplicar funcao -----------------
   if (munis == "all") {
     
-    x = munis_df$abrev_muni
+    # seleciona todos municipios ou RMs do ano escolhido
+    x = munis_list$munis_metro[ano_metro == ano]$abrev_muni
     
   } else (x = munis)
   
   # Parallel processing using future.apply
   # purrr::walk(x, merge_renda_setores)
-  future::plan(future::multiprocess)
+  future::plan(future::multicore)
```

Comment:
@kauebraga @mvpsaraiva 

`future::multicore()` não funciona no Windows, até onde eu sei só em Linux e MacOS.

Retirado da documentação (https://www.rdocumentation.org/packages/future/versions/1.21.0/topics/multicore):

> Not all operating systems support process forking and thereby not multicore futures. For instance, forking is not supported on Microsoft Windows. Moreover, process forking may break some R environments such as RStudio. Because of this, the future package disables process forking also in such cases. See supportsMulticore() for details. Trying to create multicore futures on non-supported systems or when forking is disabled will result in multicore futures falling back to becoming sequential futures.

Pra Windows tem que usar a `future::multisession()` (https://www.rdocumentation.org/packages/future/versions/1.21.0/topics/multisession).

Essa `future::multiprocess()` era usada pra escolher automaticamente entre a multicore a multisession de forma automática, a depender do sistema operacional que o código estivesse rodando. Mas hoje tá depreciada. A documentação também acaba sugerindo usar a multisession:

> WARNING: Consider the 'multiprocess' future plan deprecated. Instead, explicitly specify 'multisession' or 'multicore'. The former works everywhere and is the recommended one between the two. Forked processing, which 'multicore' uses, is unstable in various environment and setups. The 'multiprocess' alias is therefore being phased out.